### DISCUSSION TITLE
V3 createReceiptShipment method: 'Carrier Name' input for case of Etsy untracked carrier ('Other')

### DISCUSSION THREAD

**Discussion by duongvd2106** [etsy/open-api#1091] [Q&A]
Hi there,

Some of our customers have asked to submit tracking information for orders which have shipping carriers that are not on Etsy's "Shipping Carrier" dropdown list (e.g., Evri, Bpost, Fastway). We plan to send a POST createReceiptShipment request to Etsy V3, with the order's tracking_code and carrier_name = 'other'. However, I have a question.

When our customers (Etsy sellers) ship these orders using Etsy's app, if they select "Other" for the "Shipping carrier", there is an additional field called "Shipping carrier" where they can enter the name of the shipping carrier. 

![other_tracking_carrier_name](https://github.com/etsy/open-api/assets/13653172/3608afa6-4c1e-4a46-9498-8ab0b5f5ef05)

Is there a similar parameter in the API? Do we need to take care of this? How will the system handle it if we POST with carrier_name = 'other'?

I apologize for the question, but I couldn't find much information on this topic. Thank you for your time.

**Comment by emergingdzns** [comment]
I *think* you can just use "other" as the carrier name. But I am not sure to be honest. I'd say give it a try and see if you get an error back.

**Reply by duongvd2106** [reply]
Thank you, I will do this.
